apis/externalsecrets/v1alpha1/secretstore_types.go

```diff
@@ -106,6 +109,24 @@ type SecretStoreList struct {
 	Items           []SecretStore `json:"items"`
 }
 
-func init() {
-	SchemeBuilder.Register(&SecretStore{}, &SecretStoreList{})
+// +kubebuilder:object:root=true
+
+// ClusterSecretStore represents a secure external location for storing secrets, which can be referenced as part of `storeRef` fields.
+// +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster,categories={externalsecrets},shortName=css
+type ClusterSecretStore struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec SecretStoreSpec `json:"spec,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+
+// ClusterSecretStoreList contains a list of ClusterSecretStore.
+type ClusterSecretStoreList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []ExternalSecret `json:"items"`
```

Comment:
Hey mate! I believe the `Items` here should be `[]ClusterSecretStore` 🤔 

What do you think?

Reply:
Good catch, fixed.